keps/sig-node/3695-pod-resources-for-dra/kep.yaml

```diff
@@ -18,12 +18,12 @@ see-also:
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.27"
+latest-milestone: "v1.34"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
```

Comment:
Line 31 below needs an update.
```
  beta: "v1.33"
```

Reply:
I referred the other PRs and looking at them i specifically dint change it as i thought this is enough to reflect the current status `latest-milestone: "v1.34"`